app/controllers/teams_controller.rb

```diff
@@ -44,7 +44,7 @@ def typeahead
     authorize @team
     @query = params[:query]
     matches = User.search_from_query(@team.member_ids, "#{@query}%").pluck(:username)
-    matches = matches.map { |user| { username: user } }
+    matches = matches.map { |user| { name: user } }
```

Comment:
I rewrote the javascript in teams.coffee. Now I use one function for both forms.